Examples/Algorithms/Generators/ActsExamples/Generators/VertexGenerators.hpp

```diff
@@ -46,4 +46,40 @@ struct GaussianPrimaryVertexPositionGenerator
   }
 };
 
+//
+struct GaussianDisplacedVertexPositionGenerator
+    : public EventGenerator::PrimaryVertexPositionGenerator {
+  Acts::Vector4 stddev = {0.0, 0.0, 0.0, 0.0};
+  Acts::Vector4 mean = {0.0, 0.0, 0.0, 0.0};
+
+  Acts::Vector4 operator()(RandomEngine& rng) const override {
+    double min_value = -2 * M_PI;  // -π
+    double max_value = 2 * M_PI;   // π
```

Comment:
You're getting random numbers between $[-2\pi, +2\pi]$ this way, but I think you want $[-\pi,\pi]$ to get one full circle.